examples/event_handler_rest/src/accessing_request_details_headers.py

```diff
@@ -16,7 +16,7 @@
 def get_todos():
     endpoint = "https://jsonplaceholder.typicode.com/todos"
 
-    api_key: str = app.current_event.get_header_value(name="X-Api-Key", case_sensitive=True, default_value="")
+    api_key = app.current_event.headers["X-Api-Key"]
```

Comment:
How about using `.get()`? So we prevent the customer from having problems with `KeyError`.